doc/en/monkeypatch.rst

```diff
@@ -46,10 +46,13 @@ environment variable is missing, or to set multiple values to a known variable.
 :py:meth:`monkeypatch.setenv` and :py:meth:`monkeypatch.delenv` can be used for
 these patches.
 
-4. Use :py:meth:`monkeypatch.syspath_prepend` to modify the system ``$PATH`` safely, and
+4. Use ``monkeypatch.setenv("PATH", value, prepend=True)`` to modify ``$PATH``, and
```

Comment:
@nicoddemus woops this should be `prepend=":"`

Reply:
@graingert 
Can you do a followup PR then, please?

Reply:
Already done